managed/src/main/java/com/yugabyte/yw/models/InstanceType.java

```diff
@@ -179,6 +190,9 @@ public static void deleteInstanceTypesForProvider(Provider provider) {
       entries = entries.stream()
```

Comment:
Instead consider moving the whole code (line 190 to 196) in populateDefaultIfEmpty() 
And name the method populateDefaultsIfEmpty (plural)